sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/implementation/oauth2/AbstractOAuth2AuthorizationCodeGrantRequestEntityConverter.java

```diff
@@ -20,30 +19,26 @@
 public abstract class AbstractOAuth2AuthorizationCodeGrantRequestEntityConverter
     extends OAuth2AuthorizationCodeGrantRequestEntityConverter {
 
+    private static final MultiValueMap<String, String> EMPTY_MULTI_VALUE_MAP =
+            new MultiValueMapAdapter<>(Collections.emptyMap());
+
+    protected AbstractOAuth2AuthorizationCodeGrantRequestEntityConverter() {
+        addHeadersConverter(this::getHttpHeaders);
+        addParametersConverter(this::getHttpBody);
+    }
```

Comment:
We should explicitly use this default constructor in the subclasses  
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/implementation/webapp/AadOAuth2AuthorizationCodeGrantRequestEntityConverter.java#L34-L36

Reply:
Not necessary.
Refs:
https://stackoverflow.com/questions/2054022/is-it-unnecessary-to-put-super-in-constructor

Test:

![image](https://user-images.githubusercontent.com/13167207/196083442-404689d8-3df4-4921-b524-7fccf96a70e6.png)